workload_runner.py

```diff
@@ -121,17 +121,21 @@ def main() -> None:
 
     dispatch = event.Dispatcher()
     dispatch.add(*osio.resume(CLI_ARGS.namespace))
-    dispatch.add(osio.start(namespace=CLI_ARGS.namespace,
-                            storage_class=CLI_ARGS.storageclass,
-                            access_mode=CLI_ARGS.accessmode,
-                            interarrival=CLI_ARGS.osio_interarrival,
-                            lifetime=CLI_ARGS.osio_lifetime,
-                            active=CLI_ARGS.osio_active_time,
-                            idle=CLI_ARGS.osio_idle_time,
-                            kernel_slots=CLI_ARGS.osio_kernel_slots,
-                            kernel_untar=CLI_ARGS.osio_kernel_untar,
-                            kernel_rm=CLI_ARGS.osio_kernel_rm,
-                            workload_image=CLI_ARGS.osio_image))
+    events = osio.start(namespace=CLI_ARGS.namespace,
+                        storage_classes=CLI_ARGS.storageclasses,
+                        access_mode=CLI_ARGS.accessmode,
+                        interarrival=CLI_ARGS.osio_interarrival,
+                        lifetime=CLI_ARGS.osio_lifetime,
+                        active=CLI_ARGS.osio_active_time,
+                        idle=CLI_ARGS.osio_idle_time,
+                        kernel_slots=CLI_ARGS.osio_kernel_slots,
+                        kernel_untar=CLI_ARGS.osio_kernel_untar,
+                        kernel_rm=CLI_ARGS.osio_kernel_rm,
+                        workload_image=CLI_ARGS.osio_image)
+
+    for event_item in events:
+        dispatch.add(event_item)
+
```

Comment:
It looks like dispatch.add() can take a list of events already: https://github.com/red-hat-storage/ocs-monkey/blob/3d2bf9f96d75de4a70a5cd52d477d476149cc664/event.py#L58
This change may not be necessary. (See L123, above for resume which is a list of events)

Reply:
I missed that, thanks. Updated the code.